### DISCUSSION TITLE
Suggestion: one more reason for dnglab instead of the Adobe converter

### DISCUSSION THREAD

**Discussion by Kadigan** [dnglab/dnglab#211] [Ideas]
The Adobe DNG converter has an OS support list, and macOS 10.12 is _not on it_ any more. If you happen to be on macOS 10.12 (or older), `dnglab` is **the only tool** that supports future formats like CR3 or RAF.

**Comment by sagz** [comment]
How do you get a macos build of dnglab?

**Comment by Kadigan** [comment]
`make && sudo make install`? :D

But seriously, you have the build instructions on the main page. It requires `rust` (specifically, `cargo`) which can be a bit of a pain to get going... but otherwise, pretty straightforward.
